packages/terra-button/src/Button.jsx

```diff
@@ -15,7 +15,7 @@ const KEYCODES = {
 const ButtonVariants = {
   NEUTRAL: 'neutral',
   EMPHASIS: 'emphasis',
-  'DE-EMPSHASIS': 'de-emphasis',
+  'DE-EMPHASIS': 'de-emphasis',
```

Comment:
The linked issue mentions leaving the original misspelling for passivity and adding the correct spelling as a new entry

Reply:
If the existing is left, should we add a TODO for later removal with a major version bump?

Reply:
I'm in favor of aliasing the miss-spelling and keeping the api passive until the next major version bump.

Reply:
Done cb01ea2